src/node_report_utils.cc

```diff
@@ -110,65 +110,28 @@ void ReportPath(uv_handle_t* h, std::ostringstream& out) {
 
 // Utility function to walk libuv handles.
 void WalkHandle(uv_handle_t* h, void* arg) {
-  std::string type;
+  const char* type = uv_handle_type_name(h->type);
   std::ostringstream data;
   JSONWriter* writer = static_cast<JSONWriter*>(arg);
   uv_any_handle* handle = reinterpret_cast<uv_any_handle*>(h);
 
   // List all the types so we get a compile warning if we've missed one,
```

Comment:
This comment can probably be removed now, right?

Reply:
:+1: Thanks, done!